ping.py

```diff
@@ -0,0 +1,26 @@
+import os
+import time
+
+from sqlalchemy import create_engine
+from sqlalchemy.engine.url import make_url
+
+from CTFd.config import Config
+
+url = make_url(Config.DATABASE_URL)
+if not url.drivername.startswith("mysql"):
+    exit(0)
+
+engine = create_engine(url)
+print(f"Waiting for {url.host}:{url.port or 3306} to be ready")
+
+while True:
+    try:
+        engine.raw_connection().ping()
+        print()
+        break
+    except:
```

Comment:
Is there a more specific error that can be caught here?

Reply:
I'm okay with this being a catch all as long as this script is covered under flake8 and you ignore the flake8 lint.